### PR TITLE
Fix a number of typescript issues

### PR DESCRIPTION
**Pull request by silverwind**
Fixes 69 typescript errors found in the `admin` and `markup` folders.

### CODE REVIEW

**Review by yardenshoham — APPROVED** [review]

File: web_src/js/markup/html2markdown.ts [hunk 1]
```diff
@@ -44,32 +43,29 @@ function prepareProcessors(ctx:ProcessorContext): Processors {
...
+    P(el: HTMLParagraphElement) {
       el.textContent = `${el.textContent}\n`;
     },
-    BLOCKQUOTE(el) {
+    BLOCKQUOTE(el: HTMLElement) {
       el.textContent = `${el.textContent.replace(/^/mg, '> ')}\n`;
     },
-
-    OL(el) {
+    OL(el: HTMLOListElement) {
```
Comment (wxiaoguang) [inline web_src/js/markup/html2markdown.ts]: It is also reused by `UL`

```
  processors['UL'] = processors.OL;
```

**Review by delvh — APPROVED** [review]
Apart from @wxiaoguang's comment LGTM

**Comment by wxiaoguang** [comment]
By the way, please review  Refactor sidebar label selector #32460

File: web_src/js/markup/html2markdown.ts (hunk 1 above)
Comment (silverwind) [inline web_src/js/markup/html2markdown.ts]:
```suggestion
    OL(el: HTMLElement) {
```

Comment (silverwind) [reply web_src/js/markup/html2markdown.ts]: Made it `HTMLElement`.

**Pull request merged**